std_msgs/Header header
int32 xian_dj_retractable_platform_params_server_heart_beat 
int32 xian_dj_retractable_platform_control_heart_beat 
int32 xian_dj_retractable_platform_base_electric_heart_beat 
int32 xian_dj_retractable_platform_error_monitor_heart_beat 
int32 xian_dj_retractable_platform_back_end_heart_beat 
int32 xian_dj_retractable_platform_stand_linear_actuator_enble 
int32 xian_dj_retractable_platform_stand_linear_actuator_move 
int32 xian_dj_retractable_platform_first_linear_actuator_enble 
int32 xian_dj_retractable_platform_first_linear_actuator_move 
int32 xian_dj_retractable_platform_second_linear_actuator_enble 
int32 xian_dj_retractable_platform_second_linear_actuator_move 
int32 xian_dj_retractable_platform_stand_linear_actuator_stand_cmd 
int32 xian_dj_retractable_platform_stand_linear_actuator_sit_cmd 
int32 xian_dj_retractable_platform_first_linear_actuator_up_cmd 
int32 xian_dj_retractable_platform_first_linear_actuator_down_cmd 
int32 xian_dj_retractable_platform_second_linear_actuator_up_cmd 
int32 xian_dj_retractable_platform_second_linear_actuator_down_cmd 
int32 xian_dj_retractable_platform_params_server_error 
int32 xian_dj_retractable_platform_control_error 
int32 xian_dj_retractable_platform_base_electric_error 
int32 xian_dj_retractable_platform_back_end_error 

int32 xian_dj_retractable_platform_error_code 